src/systems/user_commands/UserCommands.cc

```diff
@@ -666,6 +674,10 @@ void UserCommands::Configure(const Entity &_entity,
   this->dataPtr->node.Subscribe(lightTopic, &UserCommandsPrivate::OnCmdLight,
                                 this->dataPtr.get());
 
+  std::string materialColorTopic{"/world/" + validWorldName +"/material_color"};
```

Comment:
```suggestion
  std::string materialColorTopic{"/world/" + validWorldName + "/material_color"};
```

Reply:
So the only reason that has that format is it would break 80 chars per line and fail codecheck if I add the space there. 🤣

Want me to remove the proceeding space for symmetry?

Reply:
Sadly I can't commit this change as it breaks codecheck going to 81 lines when max is 80, so marking as resolved unless there is a different suggested change desired for it:
![image](https://github.com/gazebosim/gz-sim/assets/10233412/284eeb29-5493-498e-9f52-844d75823a0b)